app/views/directives/traffic-table.html

```diff
@@ -14,9 +14,9 @@
   <tbody ng-if="(portsByRoute | hashSize) == 0">
     <tr><td colspan="7"><em>{{emptyMessage || 'No routes or ports to show'}}</em></td></tr>
   </tbody>
-  <tbody ng-if="(portsByRoute | hashSize) > 0">
-    <tr ng-repeat-start="(routeName,ports) in portsByRoute" style="display:none;"></tr>
-    <tr ng-repeat="port in ports" ng-if="routeName !== ''">
+  <tbody ng-if="(portsByRoute | hashSize) > 0" ng-repeat-start="(routeName,ports) in portsByRoute" style="display:none;"></tbody>
+  <tbody ng-repeat="port in ports" ng-if="routeName !== ''">
```

Comment:
@spadgett 
> I think we have the 2px border on other responsive tables. Can you check? If so, let's open a separate issue to fix them all.

It seems we do have a 2px border on the `pods-table` as well as the routes table but it appears to come from the table's top border rather than the `tbody`. In the `traffic-table`, the table's top border is adding `2px`, and an additional `2px` are getting added by the first visible `tbody` due to the hidden `tbody` right before it.
 
![pods-table-mobile](https://cloud.githubusercontent.com/assets/3902875/21229716/83ce5318-c2b0-11e6-8fe7-b179b8e85d72.png)

I can go ahead and open an issue to address this by repeating the `tr` tag rather than the `tbody`, and apply this change to the pods and routes table as well.